modules/openapi-generator/src/main/resources/typescript-fetch/sagas.mustache

```diff
@@ -2,33 +2,36 @@
 /* eslint-disable */
 {{>licenseInfo}}
 
+// Import not used depending on template conditions
+// @ts-ignore
 import {Api} from './';
+// Import not used depending on template conditions
+// @ts-ignore
 import {List} from 'immutable';
+// Some imports not used depending on template conditions
+// @ts-ignore
 import {all, fork, put, takeLatest} from "redux-saga/effects";
+// Some imports not used depending on template conditions
+// @ts-ignore
 import {apiCall, createSagaAction as originalCreateSagaAction, BaseEntitySupportPayloadApiAction, BasePayloadApiAction, NormalizedRecordEntities, normalizedEntities} from "../runtimeSagasAndRecords";
+// Import not used depending on template conditions
+// @ts-ignore
```

Comment:
**question:** Is there a specific condition where `Action` is not used in the template? On a first glance it looks like a core part of a saga definition

Reply:
If ``operations`` is empty.

Reply:
Ah, I see. Wondering whether the generator should not generate this file at all, if there is no operations